pkg/kubelet/volumemanager/reconciler/reconciler.go

```diff
@@ -137,6 +138,7 @@ type reconciler struct {
 	volumePluginMgr               *volumepkg.VolumePluginMgr
 	kubeletPodsDir                string
 	timeOfLastSync                time.Time
+	operationStartTimeCache       util.OperationStartTimeCache
```

Comment:
I will need to add comments here to explain why kubelet needs a bit change.
Confirmed with @msau42 kubelet attaching/detaching workflow should be deprecated.
This newly introduced cache here should be removed when the mentioned work has been done

Reply:
added e2e test